model/modelindexer/indexer_test.go

```diff
@@ -103,14 +103,17 @@ func TestModelIndexer(t *testing.T) {
 	// Closing the indexer flushes enqueued events.
 	err = indexer.Close(context.Background())
 	require.NoError(t, err)
+	stats := indexer.Stats()
+	assert.GreaterOrEqual(t, stats.BytesTotal, int64(18500))
```

Comment:
It seems that the batch size that is written to the buffer differs between MacOSX and other OS. The event size in MacOSX is 190 bytes vs 185 in Windows and Linux. 18500 is because we're writing 100 events above.